charts/kcp/templates/kcp.yaml

```
@@ -126,9 +137,9 @@ spec:
   - ReadWriteOnce
   resources:
     requests:
-      storage: 1Gi
-  storageClassName: gp2
-  volumeMode: Filesystem
+      storage: {{ .Values.kcp.volumeSize }}
+  {{- with .Values.kcp.volumeClassName -}}storageClassName: {{ . }}{{- end }}
```

```suggestion
  {{ with .Values.kcp.volumeClassName -}}storageClassName: {{ . }}{{- end }}
```
Need to remove the leading dash, otherwise the `storageClassName` line will end up on the same line as the `volumeSize`.